WelcomeHome/WelcomeHome.DAL/Repositories/SocialPayoutRepository.cs

```diff
@@ -54,4 +69,41 @@ public async Task DeleteAsync(Guid socialPayoutId)
 		_dbContext.SocialPayouts.Remove(socialPayout);
 		await _dbContext.SaveChangesAsync().ConfigureAwait(false);
 	}
+
+
+	private async void AddStepsToSicialPayout(Dictionary<int, Step> steps, SocialPayout socialPayout)
+	{
+		foreach (var step in steps.Values) 
+		{
+			step.Id = Guid.NewGuid();
+		}
+
+		_dbContext.Steps.AddRange(steps.Values);
```

Comment:
i think social payouts will be connected with steps here and we don't need method  GeneratePaymentStep